packages/terra-functional-testing/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 3.3.0 - (July 5, 2022)
+
 * Added
   * Added a util module for calling github's API.
```

Comment:
Is the change on https://github.com/cerner/terra-toolkit/pull/752 being used anywhere? It doesn't look like it based on the PR description. Is it still worth doing a release for this change if it is not being used anywhere?